app/experimenter/templates/experiments/section_addon.html

```diff
@@ -0,0 +1,17 @@
+{% extends "experiments/section_base.html" %}
+
+{% block section_title %}
+Add-On
+{% endblock %}
+
+{% block section_content %}
+  <h4>{{ experiment.addon_name }}</h4>
+  <p>
+    <strong class="mr-2">Experiment ID:</strong>
+    <code>{{ experiment.addon_experiment_id }}</code>
+  </p>
+  <p><strong>Test Build</strong></p>
+  <p><a href="{{ experiment.addon_testing_url }}" target="_blank" rel="noreferrer noopener">{{ experiment.addon_testing_url }}</a></p>
```

Comment:
It's a bit weird to display these and assume that they're valid URLs. The fields, in the model, are nullable. 
I'd rather it was something like

```django
{% if experiment.addon_testing_url %}
  <a href="{{ experiment.addon_testing_url }}" target="_blank" rel="noreferrer noopener">{{ experiment.addon_testing_url }}</a>
{% else %}
  <i>Not set</i>
{% endif %}
```

Reply:
That's [implicit here](https://github.com/mozilla/experimenter/pull/1064/files/49e3d4a07ac0d29c36dde4c990eee0978e805c2f#diff-b19c51dfc50ef6ccf00d88b6777481c2R62) by checking `completed_addon` before we show the content of this section.